app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,20 @@
+from .dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            skill_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def player_info(self) -> str:
+        nick = self.nickname
+        skill = self._skill_level
```

Comment:
no need to create variables for class attributes, use brackets to break the string in return

Reply:
Done